### PR TITLE
chat with routes. icons to the navbar

### PR DESCRIPTION
**Pull request by DavidSurkov**
Changed chat flow through routes and fixed icons in navbar

### CODE REVIEW

File: src/pages/Chat/ChatRoom/ChatRoom.tsx [hunk 1]
```diff
@@ -83,69 +95,75 @@ function ChatRoom({
   const disableInput = (): boolean => {
     if (
       userRole === "client" &&
```
Comment (xx-TOOL-xx) [inline src/pages/Chat/ChatRoom/ChatRoom.tsx]: can user roles be enum? I believe it should be already somwhere in project, or create a new one. Don't compare to string

File: src/pages/Chat/ChatRoom/ChatRoom.tsx [hunk 2]
```diff
@@ -83,69 +95,75 @@ function ChatRoom({
...
-      roomStatus === ChatRoomStatusEnum.CLIENT_ONLY
+      room?.status === ChatRoomStatusEnum.CLIENT_ONLY
     ) {
       return false;
     }
     if (
       userRole === "freelancer" &&
```
Comment (xx-TOOL-xx) [inline src/pages/Chat/ChatRoom/ChatRoom.tsx]: same

**Review by xx-TOOL-xx — APPROVED** [review]

File: src/pages/Chat/ChatRoom/ChatRoom.tsx [hunk 3]
```diff
@@ -61,91 +74,97 @@ function ChatRoom({
...
-      userRole === "freelancer" &&
-      roomStatus === ChatRoomStatusEnum.FREELANCER_ONLY
+      userRole === UserRoleEnum.FREELANCER &&
+      room?.status === ChatRoomStatusEnum.FREELANCER_ONLY
     ) {
       return false;
     }
-    return roomStatus !== ChatRoomStatusEnum.FOR_ALL;
+    return room?.status !== ChatRoomStatusEnum.FOR_ALL;
   };
```
Comment (lyudad) [inline src/pages/Chat/ChatRoom/ChatRoom.tsx]: move everything from line 38 to 109 to custom hook

Comment (DavidSurkov) [reply src/pages/Chat/ChatRoom/ChatRoom.tsx]: done

**Pull request merged**